x/ecocredit/go.mod

```diff
@@ -18,4 +18,12 @@ require (
 
 replace github.com/regen-network/regen-ledger => ../../
```

Comment:
Trying to figure out solution for this issue:
```
go mod tidy
ai@ai-ThinkPad-T450:~/go/src/github.com/regen-network/regen-ledger/x/ecocredit$ go mod tidy
go: found github.com/regen-network/regen-ledger/orm in github.com/regen-network/regen-ledger/orm v0.0.0-00010101000000-000000000000
go: found github.com/regen-network/regen-ledger/types in github.com/regen-network/regen-ledger/types v0.0.0-00010101000000-000000000000
go: found github.com/regen-network/regen-ledger/types/module/client/cli in github.com/regen-network/regen-ledger/types v0.0.0-00010101000000-000000000000
go: found github.com/regen-network/regen-ledger/types/module/server in github.com/regen-network/regen-ledger/types v0.0.0-00010101000000-000000000000
go: found github.com/regen-network/regen-ledger/types/testutil in github.com/regen-network/regen-ledger/types/testutil v0.0.0-00010101000000-000000000000
go: found github.com/regen-network/regen-ledger/types/module in github.com/regen-network/regen-ledger/types v0.0.0-00010101000000-000000000000
go: github.com/regen-network/regen-ledger/types@v0.0.0-00010101000000-000000000000 requires
        github.com/regen-network/regen-ledger/app@v0.0.0-00010101000000-000000000000 requires
        github.com/regen-network/regen-ledger/x/data@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
```

Reply:
Is this still relevant? If not, please consider resolving the conversation